uavcan/driver/socketcan.py

```diff
@@ -29,19 +29,27 @@
 
 logger = getLogger(__name__)
 
+# CAN-FD flags for tx frames
+CANFD_FLAG_BRS = 1
 
 # Python 3.3+'s socket module has support for SocketCAN when running on Linux. Use that if possible.
```

Comment:
Is this not a perfect opportunity to drop support for Python 2.7? I was considering to do that with my next PR to this library, where I intend to *finally* add support for async API (the current callback-based API is a bit less friendly than it could be).

Reply:
Indeed. If we can drop 2.7 support this would be the way to go. It should offer some performance improvements as well since this API is more directly coupled with the Kernel.